src/patterns/question-pages/index.md.njk

```diff
@@ -22,7 +22,12 @@ You should make sure you know why you’re asking every question and only ask us
 
 To help you work out what to ask, you can carry out a [question protocol](https://www.uxmatters.com/mt/archives/2010/06/the-question-protocol-how-to-make-sure-every-form-field-is-necessary.php).
 
-If you ask for optional information, mark the labels of optional fields with ‘(optional)’. Never mark mandatory fields with asterisks.
+If you ask for optional information:
+
+- in most contexts, mark the labels of optional fields with ‘(optional)’
+- for [radios](https://design-system.service.gov.uk/components/radios/) and [checkboxes](https://design-system.service.gov.uk/components/checkboxes/), add ‘(optional)’ to the legend
+
+Never mark mandatory fields with asterisks.
```

Comment:
Ok to leave this line in, or is there any reason to remove it?

Reply:
I think it's important to be explicit about this – it comes up every now and then on support in my experience.

Reply:
Thanks @christopherthomasdesign! Let's keep it then.